src/main/java/net/wiseoldman/beans/Metric.java

```diff
@@ -18,6 +18,8 @@ public enum Metric
 	AGILITY(HiscoreSkill.AGILITY),
 	@SerializedName("alchemical_hydra")
 	ALCHEMICAL_HYDRA(HiscoreSkill.ALCHEMICAL_HYDRA),
+	@SerializedName("araxxor")
+	ALCHEMICAL_HYDRA(HiscoreSkill.ARAXXOR),
```

Comment:
This should be `ARAXXOR`, not `ALCHEMICAL_HYDRA`

Reply:
I really shouldn't do anything on mondays; this proves it >.>